module-2/exercise/.gitignore

```diff
@@ -0,0 +1,2 @@
+.*.touch
+.cache.*
```

Comment:
I'm a little confused by this - what files are you aiming to exclude here - are we expecting files with `.<something>.touch` filenames? And if so, is there a reason to prefer to exclude them in this sub-directory rather than in the top-level one?

Reply:
.*.touch is an artifact of a bash script I have that re-runs stuff based on modification time (e.g. to run doctests or otherwise run tests on save).  I'm aware I should learn how VSCode does this (but the home-made test also works alongside vim :-) )
.cache.* is vestigial—I re-used code from the marvel exercise, where I started caching the json returned since the connection through cascading proxies to get through the corporate connection was very slow and the data was invariant.  Of course the trello data is very variable and caching was inappropriate, so these files are no longer created.

I'll tidy